network/p2p/pubsub.go

```diff
@@ -194,22 +194,24 @@ func (p PeerScoreSnapshot) IsWarning() bool {
 
 	// Check overall score.
 	switch {
-	case p.Score < 0:
+	case p.Score < -1:
 		// If the overall score is negative, the peer is in warning state, it means that the peer is suspected to be
 		// misbehaving at the GossipSub level.
 		return true
 	// Check app-specific score.
-	case p.AppSpecificScore < 0:
+	case p.AppSpecificScore < -1:
 		// If the app specific score is negative, the peer is in warning state, it means that the peer behaves in a way
 		// that is not allowed by the Flow protocol.
 		return true
 	// Check IP colocation factor.
-	case p.IPColocationFactor > 0:
+	case p.IPColocationFactor > 5:
 		// If the IP colocation factor is positive, the peer is in warning state, it means that the peer is running on the
-		// same IP as another peer and is suspected to be a sybil node.
+		// same IP as another peer and is suspected to be a sybil node. For now, we set it to a high value to make sure
+		// that peers from the same operator are not marked as sybil nodes.
+		// TODO: this should be revisited once the collocation penalty is enabled. Setting it to 20 to reduce the noise; 20 is twice the threshold (defaultBehaviourPenaltyThreshold).
```

Comment:
is this comment meant for `IPColocationFactor` or `BehaviourPenalty`?

Reply:
https://github.com/onflow/flow-go/pull/4838/commits/aeaaac76422d7e661512be9344f30d507bcaba9d